OHHTTPStubs/Sources/OHHTTPStubs.m

```diff
@@ -424,16 +424,29 @@ - (void)startLoading
                 // Notify if a redirection occurred
                 if (((responseStub.statusCode > 300) && (responseStub.statusCode < 400)) && redirectLocationURL)
```

Comment:
Self Review: This is a bug as we conditional go into here if there is a redirectLocationURL, yet we need to unconditionally depend on the line NSURLRequest *redirectRequest = (NSURLRequest*)[mReq copy];